downstream/modules/platform/proc-create-controller-form-view.adoc

```diff
@@ -0,0 +1,19 @@
+[id="proc_create-controller-form-view_{context}"]
+
+= Create-{ControllerName}-form-view
+
+== Using the Form View
```

Comment:
Should this be the title of this section instead of the what's above? It doesn't seem like there should be a title and subtitle for this short procedure, unless there were going to be multiple sections under one heading. It almost reads as if there should be a set of steps for creating the form view and then one for how to use it, but I only see one set of procedures.